app/services/hyrax/qa_select_service_decorator.rb

```diff
@@ -0,0 +1,24 @@
+# OVERRIDE: Hyrax 3.4.2 fixes issues with empty authority and no implicit conversion of string into array
+
+# frozen_string_literal: true
+
+module Hyrax
+  module QaSelectServiceDecorator
+    
+    def active?(id)
+      unless authority.find(id).empty?
+        authority.find(id)&.fetch('active')
+      end
+    end
+
+    def include_current_value(value, _index, render_options, html_options)
+      unless value.blank? || active?(value)
```

Comment:
Unless and or statements always confound me, so this is my bias coming in; I believe the following is logically the same: `if value.present? && !active?(value)`. (again `unless` with `or` always confounds me).

Given the above, my read is:

"Hey, you have a controlled value that is no longer active so we'll add the `force-select` class and add some render options."

another question is how do we handle labels for non-active values?

Reply:
This is code from Hyrax and I don't want to mess with it as we need this in Production ASAP